### PR TITLE
Updated french translation

### PR DESCRIPTION
**Pull request by gege2b**
title say it all

### CODE REVIEW

**Comment by ghost** [comment]
Un Français :fr: 
Cool 😄 
J'ai tout laissé en anglais je trouve ça tellement dégueulasse sans les accents , ça gache tout je trouve 😢

**Comment by thinkyhead** [comment]
Si c'est très gênant, nous pouvons éditer `language_fr_utf8.h` aussi.

File: Marlin/src/lcd/language/language_fr.h [hunk 1]
```diff
@@ -318,8 +358,19 @@
 #define MSG_DAC_PERCENT                     _UxGT("Driver %")
 #define MSG_DAC_EEPROM_WRITE                _UxGT("DAC EEPROM sauv.")
 
+#define MSG_FILAMENT_CHANGE_HEADER_PAUSE    _UxGT("IMPR. PAUSE")
+#define MSG_FILAMENT_CHANGE_HEADER_LOAD     _UxGT("CHARGER FIL")
+#define MSG_FILAMENT_CHANGE_HEADER_UNLOAD   _UxGT("DECHARGER FIL")
+#define MSG_FILAMENT_CHANGE_OPTION_HEADER   _UxGT("OPTIONS REPRISE:")
+#define MSG_FILAMENT_CHANGE_OPTION_PURGE    _UxGT("Purger encore")
+#define MSG_FILAMENT_CHANGE_OPTION_RESUME   _UxGT("Reprendre")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_fr.h]: This is a duplicate of the one below "Reprendre impr."

**Comment by thinkyhead** [comment]
I've also added the new strings to `language_fr_utf8.h`. Thanks for the update!

**Comment by gege2b** [comment]
damn, I didn't even see the utf8 file, sorry about that and for the additional work
and thank for the merge

**Pull request merged**